src/main/java/org/fcrepo/migration/handlers/ocfl/ArchiveGroupHandler.java

```diff
@@ -246,6 +256,15 @@ public void processObjectVersions(final Iterable<ObjectVersionReference> version
                     dsCreateDates.put(dsId, dv.getCreated());
                     datastreamStates.put(f6DsId, dv.getDatastreamInfo().getState());
                 }
+
+                headOnlyStates.put(f6DsId, HEAD_ONLY_OK);
+                final var skip = headOnlyDatastreamManager.accept(dsId);
```

Comment:
Perhaps this method should be called something more descriptive like `shouldMigrateOnlyHead`. Personally, I find the generic `accept` makes it hard to interpret the meaning of the return value.